docs/Policies/NuGet-FAQ.md

```diff
@@ -168,7 +168,7 @@ This is not an issue when using PackageReference, as each project file contains
 
 **Is it possible to reserve names for packages that will be published in future?**
 
-No. If you feel that an existing package has taken the name which suits your package better, try contacting the owner of the package (select on the owner name on the package page on nuget.org). If you don't get a response within a reasonable time, [contact support](https://www.nuget.org/policies/Contact) and we'll look into it.
+Yes. You can reserve IDs for packages on [nuget.org](https://www.nuget.org/) by requesting a package ID prefix for your account. In order to request a package ID prefix, send mail to account@nuget.org with the package owner display name, and the requested package ID prefix.  
```

Comment:
Can we make the email link smart such that the subject line is prepopulated and some content is already in the body of the email and such.

Reply:
Do you know how to  do that?